cpp/ql/test/library-tests/valuenumbering/GlobalValueNumbering/ir_gvn.expected

```diff
@@ -7,61 +7,61 @@ test.cpp:
 #    1|     mu1_3(unknown)             = UnmodeledDefinition      : 
 #    1|         valnum = unique
 #    1|     r1_4(glval<int>)           = VariableAddress[p0]      : 
-#    1|         valnum = r1_4
+#    1|         valnum = GVN
 #    1|     m1_5(int)                  = InitializeParameter[p0]  : &:r1_4
-#    1|         valnum = m1_5
+#    1|         valnum = GVN
```

Comment:
It looks like this test has become useless. It'll need to call something else than `toString` to produce proper debug output.

Reply:
Agreed. I've added a `getDebugString` predicate (which was the old `toString`):
```
final string getDebugString() {
    result = "ValueNumber: " +
        strictconcat(this.getAnInstruction().getUnconvertedResultExpression().toString(), ", ")
  }
```

which might be more appropriate.